docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1619,359 +1635,597 @@ We **strongly recommend** using these markers any time when you use an extended
+
+> For applying the `:remove()` pseudo-class to any element, a [universal selector](https://www.w3.org/TR/selectors-4/#the-universal-selector) `*` should be used. Otherwise such extended selector may be considered as invalid, e.g. `.banner > :remove()` is not valid for removing any child element of `banner` class element, so it should look like `.banner > *:remove()`.
+
+> If the `:remove()` pseudo-class or the `remove` pseudo-property is used, all style properties are ignored except of the [`debug` pseudo-property](#selectors-debug-mode).
+
 
 **Examples**
 ```
-div.inner:remove()
+div.banner:remove()
 div:has(> div[ad-attr]):remove()
-div:xpath(../..):remove()
-div:contains(target text) { remove: true; }
-div[class]:has(> a:not([id])) { remove: true; }
+
+div:contains(advertisement) { remove: true; }
+div[class]:has(> a > img) { remove: true; }
 ```
 
-> All style properties will be ignored if `:remove()` pseudo-class or `remove` pseudo-property is used.
+> **Note**
+>
+> Rules with the `remove` pseudo-property should use `#$?#` marker: `$` for CSS style rules syntax, `?` for ExtendedCss syntax.
+
+
+#### Pseudo-class `:is()` {#extended-css-is}
+
+The `:is()` pseudo-class allows to match any element that can be selected by any of selectors passed to it. Invalid selectors are skipped and the pseudo-class deals with valid ones with no error thrown. Our implementation of the [native `:is()` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:is).
+
+**Syntax**
+
+```
+[target]:is(selectors)
+```
+- `target` — optional, standard or extended css selector, can be missed for checking *any* element
+- `selectors` — [*forgiving selector list*](https://drafts.csswg.org/selectors-4/#typedef-forgiving-selector-list) of standard or extended selectors. For extended selectors only compound selectors are supported, not complex.
+
+##### `:is()` limitations {#extended-css-is-limitations}
+
+> Rules with the `:is()` pseudo-class should use the [native implementation of `:is()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) if rules use `##` marker and it is possible, i.e. with no other extended selectors inside. To force applying ExtendedCss rules with `:is()`, use `#?#`/`#$?#` marker explicitly.
+
+> If the `:is()` pseudo-class argument `selectors` is an extended selectors, due to the way how the `:is()` pseudo-class is implemented in ExtendedCss v2.0, it is impossible to apply it to the top DOM node which is `html`, i.e. `#?#html:is(<extended-selectors>)` does not work. So if `target` is not defined or defined as a [universal selector](https://www.w3.org/TR/selectors-4/#the-universal-selector) `*`, the extended pseudo-class applying is limited to **`html`'s children**, e.g. rules `#?#:is(...)` and `#?#*:is(...)` are parsed as `#?#html *:is(...)`. Please note that there is no such limitation for a standard selector arg, i.e. `#?#html:is(.locked)` works fine.
```

Comment:
is an extended selectors -> is an extended selector

Reply:
>standard selector arg

Is it some kind of a slang for `argument`? Should we use it in the full form? If yes, there are other examples of this throughout the text that should be fixed too